src/nfa/nfa.cc

```diff
@@ -740,12 +828,12 @@ Nfa Mata::Nfa::remove_epsilon(const Nfa& aut, Symbol epsilon) {
     while (changed) { // compute the fixpoint
         changed = false;
         for (size_t i=0; i < num_of_states; ++i) {
-            const auto& state_transitions{ aut.delta[i] };
-            const auto state_symbol_transitions{
-                state_transitions.find(Move{epsilon}) };
-            if (state_symbol_transitions != state_transitions.end()) {
+            const Post& post{ aut.delta[i] };
+            const auto emove{
```

Comment:
I would rename `emove` to `eps_move`, so it is more obvious.

Reply:
Done.